curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655235c2e607297f00316650.md

```diff
@@ -7,13 +7,13 @@ dashedName: step-35
 
 # --description--
 
-Next, use the `classList` property and the `add()` method to add the `playing` class to the `playButton` element. This will look for the class `playing` in the CSS file and add it to the `playButton` element.
+Next, use the `classList` property and the `add()` method to add the `"playing"` class to the `playButton` element. This will look for the class `playing` in the CSS file and add it to the `playButton` element.
```

Comment:
```suggestion
Next, use the `classList` property and the `add()` method to add the `"playing"` class to the `playButton` element. This will look for the class `"playing"` in the CSS file and add it to the `playButton` element.
```